drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/RuleModellerPageView.java

```diff
@@ -35,21 +38,34 @@
     @DataField("rule-modeller-container")
     private Div ruleModellerContainer;
 
+    @DataField("description-heading")
+    private Heading descriptionHeading;
+
     private RuleModellerPage page;
 
     @Inject
-    public RuleModellerPageView(final Div ruleModellerContainer) {
+    public RuleModellerPageView(final Div ruleModellerContainer,
+                                final @Named("h5") Heading descriptionHeading) {
         this.ruleModellerContainer = ruleModellerContainer;
+        this.descriptionHeading = descriptionHeading;
     }
 
     @Override
     public void init(final RuleModellerPage page) {
         this.page = page;
+        if (page.plugin() instanceof BRLActionColumnPlugin) {
+
```

Comment:
If it is then do what exactly? If we don't want to do anything do we need this ```if(..)``` block at all?